app/Directory.Packages.props

```diff
@@ -1,6 +1,7 @@
 <Project>
   <PropertyGroup>
     <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
+    <MauiVersion>8.0.6</MauiVersion>
```

Comment:
Maybe we can just use normal version style and not a property?

Reply:
`MauiVersion` is used 3 times below, so (unless I'm missing something) a property seems better, right?